cluster/mesos/docker/certs/rebuild.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
```

Comment:
looks like a bunch of these got 2014 in them... was the auto-gen script updated recently?